test/js/test.js

```diff
@@ -152,10 +157,23 @@ saveFiles
 */
 
 function main() {
+  inquirer.prompt(questions.files, makeKeyPair);
+}
+
+function makeKeyPair(answers) {
+  state.certFile = answers.certFile;
+  state.keyFile = answers.keyFile;
   console.log("Generating key pair...");
-  state.keyPair = crypto.generateKeyPair(state.keyPairBits);
-  console.log();
-  inquirer.prompt(questions.email, register)
+  child_process.exec("openssl req -newkey rsa:2048 -keyout " + state.keyFile + " -days 3650 -subj /CN=example.com -nodes -x509 -out temp-cert.pem", function (error, stdout, stderr) {
```

Comment:
Should example.com be hardcoded?  If it is intended... it might be good to include comments at the top of the file or a README regarding the necessary setup to get this to work.